### DISCUSSION TITLE
All users that connect to Trino with the JDBC driver must be granted access to query tables, how can I do that?

### DISCUSSION THREAD

**Discussion by BorderCloud** [trinodb/trino#19108] [Q&A] [answered]
I have questions about the requirement "[All users that connect to Trino with the JDBC driver must be granted access to query tables in the system.jdbc schema](https://trino.io/docs/current/client/jdbc.html#registering-and-configuring-the-driver)" in the jdbc doc of Trino.

How to allow a user to connect to Trino via the JDBC driver?
In other words... Have you examples ? to write in the Trino console or in a configuration file to:
- create a new user with his password
- Allow this user to grant access to tables in a precise schema via JDBC

It's maybe too trivial... But I don't find a simple example/tutorial. 
And do you know how can I debug a connection problem via JDBC? (a version of JDBC in mode debug ?)
Thanks.

**Comment by hashhar** [comment, marked as answer]
By default you don't need to do anything since every user has access to all tables.

That documentation calls it out because a lot of times people enable some form of access control and disable access to the `system` catalog entirely which breaks tools which provide a schema explorer since it needs to access tables under `system.jdbc` schema.

So unless you've installed some access control you don't need to do anything.
